lua/ttt2/libraries/entspawnscript.lua

```diff
@@ -119,6 +118,17 @@ local spawnData = {
 	}
 }
 
+local kindToSpawnType = {
+	[WEAPON_PISTOL] = WEAPON_TYPE_PISTOL,
+	[WEAPON_HEAVY] = WEAPON_TYPE_ASSAULT,
```

Comment:
should I rename "ASSAULT" to "HEAVY"?

Reply:
Yeah I guess keeping that could make sense, I'd prefer RIFLE in general or AUTOMATIC, but the old is there, so might just use it. Normally there would be smgs and rifles (assault rifles and machine guns) but heavy should fit as well.